cityofaustin/atd-moped
# Data Tracker sync continued

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/15550

This adds a helper script to evaluate the sync between Moped projects and Data Tracker `projects` table and delete Knack records that have no connection to a Moped project and no connection to other Knack records.

Also, this is a tool for the toolbox. It is not going to be part of the ETL.

## Testing
**URL to test:** 

n/a

**Steps to test:**
Hard and not really useful to test this on staging so I'm going to say the best way is to check out the code. It is unlikely that we will have to run this ever again.

---
#### Ship list
